src/callbacks_step/analysis_surface_integral_2d.jl

```diff
@@ -61,13 +65,25 @@ struct DragCoefficientPressure{RealT <: Real}
     force_state::ForceState{RealT}
 end
 
+# Abstract base type used for dispatch of `analyze` for quantities 
+# requiring gradients of the velocity field.
+abstract type VariableViscous end
+
+struct LiftCoefficientShearStress{RealT <: Real} <: VariableViscous
+    force_state::ForceState{RealT}
+end
+
+struct DragCoefficientShearStress{RealT <: Real} <: VariableViscous
+    force_state::ForceState{RealT}
+end
+
 """
-    LiftCoefficientPressure(aoa, rhoinf, uinf, l_inf)
+    LiftCoefficientPressure(aoa, rhoinf, uinf, linf)
 
 Compute the lift coefficient
 ```math
 C_{L,p} \\coloneqq \\frac{\\oint_{\\partial \\Omega} p \\boldsymbol n \\cdot \\psi_L \\, \\mathrm{d} S}
-                        {0.5 \\cdot \\rho_{\\infty} \\cdot U_{\\infty}^2 \\cdot L_{\\infty}}
```

Comment:
I removed the dots to avoid confusion as we use the proper meaning of the dot in this formula as well.